readme.md

```diff
@@ -62,6 +63,9 @@
 - [`traph.macro`](https://github.com/caesarsol/traph.macro): Transform Objects easily, leveraging object getters and graphs
 - [`idx.macro`](https://github.com/dralletje/idx.macro): Traverse properties on objects and arrays
 
+### Typescript
```

Comment:
Please use the correct capitalization (`TypeScript`) here as well as in the table of contents.

If we're adding a TS section we might as well move `ts-nameof` in there from the General section.